chaqer.py

```diff
@@ -0,0 +1,389 @@
+        global headers
+        global KEY
+        params = urllib.urlencode({
+            'personGroupId' : groupID
+        })
+        try:
+            conn = httplib.HTTPSConnection(CONN_CODE)
+            conn.request("DELETE", "/face/v1.0/persongroups/{personGroupId}?%s" % params, "{body}", headers)
+            response = conn.getresponse()
+            data = response.read()
+            print(data)
+            fileName = '/tmp/' + groupID.lower() + '.txt'
+            os.remove('%s'%fileName)
+            responseFileName = '/tmp/' + 'imagesSearchedOn' + groupID.upper() + '.txt'
+            os.remove('%s'%responseFileName)
+            conn.close()
+        except Exception as e:
+            print("[Errno {0}] {1}".format(e.errno, e.strerror))
+
+
+
+
+
+    def listGroups(self):
+        global CONN_CODE
+        global headers
+        global KEY
+        params = urllib.urlencode({
+        })
+
+        try:
+            conn = httplib.HTTPSConnection(CONN_CODE)
+            conn.request("GET", "/face/v1.0/persongroups?%s" % params, "{body}", headers)
+            response = conn.getresponse()
+            data = response.read()
+            data = json.loads(data)
+            for i in range(0,len(data)):
+                print '\n\nGroupID: ' + str(data[i]["personGroupId"])
+                print 'Group Name: ' + str(data[i]["name"])
+                print 'Group Info: ' + str(data[i]["userData"])
+            conn.close()
+        except Exception as e:
+            print("[Errno {0}] {1}".format(e.errno, e.strerror))
+
+
+
+
+    def createPersonGroup(self,groupID,groupName="",groupInfo=""):
+        global CONN_CODE
+        global headers
+        global KEY
+        global NameID
+        emptyDict = {}
+        groupID = str(groupID)
+        params = urllib.urlencode({
+            'personGroupId' : groupID
+        })
+
+        body = "{'name': '%s' ,'userData':'%s'}" %(groupName,groupInfo)
+        try:
+            conn = httplib.HTTPSConnection(CONN_CODE)
+            conn.request("PUT", "/face/v1.0/persongroups/{personGroupId}?%s" % params, body, headers)
+            response = conn.getresponse()
+            data = response.read()
+            if data != '':
+                print '\n\n\n' + data
+                return
+            print '\n\n\nSuccesfully Created Group \n' + 'Group ID: ' + groupID + '\nGroup Name: ' + groupName + '\nGroup Info: ' + groupInfo
+            print data
+            conn.close()
+            responseFileName = '/tmp/' + 'imagesSearchedOn' + groupID.upper() + '.txt'
+            e = open('%s'%responseFileName,'w+')
+            e.close()
+            fileName = '/tmp/' + groupID.lower() + '.txt'
+            f = open('%s'%fileName,'w+')
+            f.close()
+        except Exception as e:
+            print e
+            print("[Errno {0}] {1}".format(e.errno, e.strerror))
+
+
+
+
+
+    def createPerson(self,groupID,personName,personInfo=""):
+        global NameID
```

Comment:
why are creating `global` variables? Maybe I am misunderstanding this, but if this is actually creating a `global` variable that can be read by other parts of the code, then this is a bad idea. 

We should have an _init_ function that sets these values for the entire class.

```
class Chaqer(object):
    def __init__(self, name, conn_code):
        self.NameID = name
        self.CONN_CODE = conn_code
        ...
```

This will allow us to have local variables for each instance of Chaqer().

```
chaqer1 = new Chaqer('shaq', 'connection', ...)
chaqer2 = new Chaqer('chuck', 'connection'. ...)
```

Then we can call each without fear of other variables bleeding into another one.